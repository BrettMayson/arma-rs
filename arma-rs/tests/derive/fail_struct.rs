use arma_rs_proc::Arma;

#[derive(Arma)]
pub struct UnitDeriveTest;

#[derive(Arma)]
pub struct EmptyDeriveTest();

pub fn main() {}
