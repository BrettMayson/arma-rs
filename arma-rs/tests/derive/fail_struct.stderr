error: Unit structs aren't supported
 --> tests/derive/fail_struct.rs:3:10
  |
3 | #[derive(Arma)]
  |          ^^^^
  |
  = note: this error originates in the derive macro `Arma` (in Nightly builds, run with -Z macro-backtrace for more info)

error: Structs with no fields aren't supported
 --> tests/derive/fail_struct.rs:6:10
  |
6 | #[derive(Arma)]
  |          ^^^^
  |
  = note: this error originates in the derive macro `Arma` (in Nightly builds, run with -Z macro-backtrace for more info)
